common.lic

```diff
@@ -471,6 +471,18 @@ module DRC
     end
   end
 
+  def check_listening
+    return if DRRoom.pcs.empty?
+    classes = assess_teach
+    @last_teacher = classes
```

Comment:
common isn't a class with instance variables. Need to remove the `@`

to confirm, `assess_teach` is a method defined in common or drinfomon? I'm unfamiliar with it either way.

Reply:
Agreed re: removing the instance vars. I suggest `@last_teacher` be owned by the calling script and returned from `check_listening`. The same with @listen_timer`.

Reply:
Thanks for the feedback. Updated the def.